docs/products/metrics/howto/migrate-influxdb-thanos.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Migrate Aiven for InfluxDB to Aiven for Metrics
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Migrate your Aiven for InfluxDB databases to Aiven for Thanos Metrics using the Aiven Console migration tool.
+
+## Compatibility considerations
+
+Before migrating an InfluxDB database to Aiven for Metrics, carefully
+review your current InfluxDB setup.
+
+- **Review database setup:** Examine your InfluxDB database's data structures, storage
+  patterns, and configurations. Identify any unique features, custom settings, and
+  specific configurations.
+- **API and query language compatibility:** Thanos Metrics uses PromQL, which is
+  different from InfluxDB's InfluxQL. Update your queries to be
+  compatible with PromQL. For assistance, you can use tools such as the
+  [InfluxQL to PromQL converter](https://github.com/logzio/influxql-to-promql-converter)
+  or Aiven's own [dashboard converter](https://github.com/Aiven-Open/influxql-to-m3-dashboard-converter).
+  The built-in dashboards provided by Aiven services through integrations will not
+  require any manual changes.
+- **Version compatibility:** Ensure that your InfluxDB version supports migration to
+  Thanos Metrics. You might need to upgrade your InfluxDB to a compatible version.
+
+## Prerequisites
+
+Before starting the migration from an Aiven for InfluxDB service:
+
+- Make a note of the Aiven project and Aiven for InfluxDB service name
```

Comment:
Maybe just add this to the sentence above since there's only one bullet point here?